pkg/users/client.go

```diff
@@ -956,3 +960,41 @@ func (c *Client) EnrollAuthFactor(ctx context.Context, opts EnrollAuthFactorOpts
 
 	return body, err
 }
+
+// ListAuthFactors lists the available authentication factors for the user.
+func (c *Client) ListAuthFactors(ctx context.Context, opts ListAuthFactorsOpts) ([]mfa.Factor, error) {
```

Comment:
Is the plan for these factors always to be the same (if so, I wonder if they should move to common in v3).

Reply:
It makes sense. I'm following what they did on Node SDK.